src/mapping/config/MappingConfiguration.hpp

```diff
@@ -49,6 +49,14 @@ class MappingConfiguration : public xml::XMLTag::Listener {
     bool         enableUnifiedMemory = false;
   };
 
+  struct GreedyParameter {
+    std::string  executor      = "reference-executor";
```

Comment:
The reference executor is essentially a CPU reference executor for Ginkgo-based implementations. For the `Eigen` backend, we typically set here `cpu` (also part of the xml-attribute list).